nautobot_floor_plan/templates/nautobot_floor_plan/floorplantile_retrieve.html

```diff
@@ -37,10 +37,18 @@
                     </span>
                 </td>
             </tr>
+            <tr>
+                <td>Tile Type</td>
+                <td>{{ object.allocation_type }}</td>
```

Comment:
Possibly:

```suggestion
                <td>{{ object.get_allocation_type_display|placeholder }}</td>
```

since this field is blankable.